frontend/src/utils/account-send.ts

```diff
@@ -178,20 +179,25 @@ export const storeSend = async ({
       txHash,
     },
   ]);
-  await localforage.setItem(`${LOCALFORAGE_ACCOUNT_SEND_KEY_PREFIX}-count-${senderAddress}-${chainId}`, count + 1);
+  await localforage.setItem(
+    `${LOCALFORAGE_ACCOUNT_SEND_KEY_PREFIX}-count-${senderAddress}-${chainId}`,
+    BigNumber.from(count).add(1).toString()
+  );
 };
 
 export const fetchAccountSends = async ({ address, viewingPrivateKey, chainId }: FetchAccountSendArgs) => {
   const localStorageKey = `${LOCALFORAGE_ACCOUNT_SEND_KEY_PREFIX}-${address}-${chainId}`;
   const values = ((await localforage.getItem(localStorageKey)) as AccountSendDataWithEncryptedFields[]) || [];
+  const localStorageCountKey = `${LOCALFORAGE_ACCOUNT_SEND_KEY_PREFIX}-count-${address}-${chainId}`;
+  const encryptionCount = (await localforage.getItem(localStorageCountKey)) as string;
 
   let accountData = [] as AccountSendData[];
   const addresses = [];
 
   for (const [index, sendInfo] of values.entries()) {
     const decryptedData = decryptData(sendInfo.accountSendCiphertext, {
       viewingPrivateKey,
-      encryptionCount: index,
+      encryptionCount: BigNumber.from(encryptionCount).add(1).toString(),
```

Comment:
https://github.com/ScopeLift/umbra-protocol/pull/494/commits/2c0807413a78b4b4f613964947433fa34853b6a3 and https://github.com/ScopeLift/umbra-protocol/pull/494/commits/84c408be28bdb90540ce03f4082206b6802b027b should solve this issue and add tests to prevent regression in the future